ingeniamotion/configuration.py

```diff
@@ -1139,3 +1174,138 @@ def get_max_current(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS)
                 f"Expected int, got {type(max_current)}"
             )
         return max_current
+
+    def get_commutation_mode(
+        self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS
+    ) -> CommutationMode:
+        """Get commutation mode in the target servo and axis.
+
+        Args:
+            servo: servo alias to reference it. ``default`` by default.
+            axis: servo axis. ``1`` by default.
+
+        Returns:
+            Commutation mode.
+
+        Raises:
+            TypeError: If some read value has a wrong type.
+
+        """
+        commutation_mode = self.mc.communication.get_register(
+            self.COMMUTATION_MODE_REGISTER, servo, axis
+        )
+        if not isinstance(commutation_mode, int):
+            raise TypeError("Commutation mode value has to be an integer")
```

Comment:
Implicit with next line

Reply:
This raise avoids mypy errors because the register value can be int, float, or str.